app/src/main/java/com/karumi/ui/domain/usecase/GetSuperHeroes.kt

```diff
@@ -0,0 +1,16 @@
+package com.karumi.ui.domain.usecase
+
+import com.karumi.ui.domain.Result
+import com.karumi.ui.domain.model.SuperHero
+
+class GetSuperHeroes {
+
+    val mockSuperHero = SuperHero(name = "Super Peter",
```

Comment:
In the original implementation, we have a simple [repository](https://github.com/Karumi/KataSuperHeroesAndroid/blob/master/app/src/main/java/com/karumi/katasuperheroes/model/SuperHeroesRepository.java) and the values it returns are private. I've read the repository will be your next PR but we should keep this as private.

Reply:
yes, this line will be dissappear in the next PR, but yes, this is a bug.